readme.md

```diff
@@ -132,12 +132,12 @@ If you need to change this in a reusable module, create a new instance:
 const ctx = new chalk.Instance({level: 0});
 ```
 
-Levels are as follows:
```

Comment:
Why did you remove this?

Reply:
It seemed like the the heading row of the table communicates the same information.

I am happy to put it back if you prefer.

Reply:
Indeed, I guess they do. I don't have a strong opinion about it so let's remove it for now unless somebody else wants to keep it.